coremltools/converters/mil/frontend/torch/ops.py

```diff
@@ -2496,3 +2496,10 @@ def std(context, node):
 def copy_(context, node):
     inputs = _get_inputs(context, node, expected=3)
     context.add(mb.identity(x=inputs[0], name=node.name))
```

Comment:
Thanks for the change, can you also add unit tests to https://github.com/apple/coremltools/blob/master/coremltools/converters/mil/frontend/torch/test/test_torch_ops.py?

Reply:
Ok, implemented tests.